src/amplitude-client.js

```diff
@@ -31,6 +31,7 @@ var AmplitudeClient = function AmplitudeClient(instanceName) {
   this._q = []; // queue for proxied functions before script load
   this._sending = false;
   this._updateScheduled = false;
+  this._onInit = [];
```

Comment:
This needs to be in the snippet right? Similar to the queued functions:
https://github.com/amplitude/Amplitude-JavaScript/blob/master/src/amplitude-snippet.js#L10-L14

The reason is that the AmplitudeClient might not be loaded yet, so you can't use it to queue 

https://github.com/amplitude/Amplitude-JavaScript/blob/master/src/index.js#L2-L11
https://github.com/amplitude/Amplitude-JavaScript/blob/40ff69f0870b43c2cd0fc97c6f7e738272ed62cf/src/amplitude-snippet.js#L7

Reply:
Good catch. Fixed.